examples/customLayers/customBilayer.cpp

```diff
@@ -70,40 +70,40 @@ extern "C" {
 
         // Manually deal with hydration for layers in
         // this example.
-        double oxSLD = (oxideHydration * *bulkOut) + ((1 - oxideHydration) * oxideSLD);
-        double headSLD = (headHydration * *bulkOut) + ((1 - headHydration) * SLDhead);
-        double tailSLD = (bilayerHydration * *bulkOut) + ((1 - bilayerHydration) * SLDtail);
+        double oxSLD = (oxideHydration * bulkOut[0]) + ((1 - oxideHydration) * oxideSLD);
```

Comment:
@Arwel the python version of this example does indexes the bulkOut with contrast number i.e `bulkOut[contrast]` but this is not the case in the C++ version. Is this intentional?

Reply:
Wrong Arwel I'm afraid, you're after @arwelHughes. Been far too long since I did any c++

Reply:
Apologies @Arwel my bad